### DISCUSSION TITLE
More Icons for SAP

### DISCUSSION THREAD

**Discussion by Aamjit** [devicons/devicon#1692] [General]
Hi all,
Would like to request for SAP icons like UI5, Fiori, SAP S4 HANA, etc. icons.

**Comment by Snailedlt** [comment]
Feel free to make issues for these (one issue for each logo). Just make sure the logos are within scope. 
See [What Icons Do We Accept?](https://github.com/devicons/devicon/wiki/What-Icons-Do-We-Accept%3F) for more info
